main.py

```diff
@@ -476,6 +476,18 @@ def add_args(parser):
         help="Multiplier of the standard deviation for the reward normalization",
     )
     args2config.update({"reward_norm_std_mult": ["gflownet", "reward_norm_std_mult"]})
+
+    parser.add_argument(
+        "--reward_func",
+        default="power",
+        type=str,
+        help="Function for rewards transformation: power or boltzmann",
+    )
+    args2config.update({"reward_func": ["gflownet", "reward_func"]})
+
+    parser.add_argument("--temperature_logits", default=1.0, type=float)
+    args2config.update({"temperature_logits": ["gflownet", "temperature_logits"]})
+
```

Comment:
params missing in the main.py / present in gflownet.py only.
With hydra we will change this anyway ;)